CODEOWNERS

```diff
@@ -150,6 +150,24 @@ k8s/**/common/family-public-law/namespace.yaml @hmcts/fpl-admins
 k8s/**/common/family-public-law/identity.yaml @hmcts/fpl-admins
 k8s/**/common/family-public-law/fpl-case-service.yaml @hmcts/fpl-admins
 
+### Adoption
+k8s/preview/common/adoption/namespace.yaml @hmcts/fpl-admins
+k8s/preview/common/adoption/identity.yaml @hmcts/fpl-admins
+k8s/preview/common/adoption/adoption-frontend.yaml @hmcts/fpl-admins
+k8s/preview/common/adoption/adoption-backend.yaml @hmcts/fpl-admins
+k8s/aat/common/adoption/namespace.yaml @hmcts/fpl-admins
```

Comment:
only aat and prod are restricted, just list aat here as no need for the others unless you want to get the code review requests automatically?

Reply:
done, narrowed to aat only